queries.sql

```diff
@@ -30,5 +30,78 @@ WHERE name != 'Gabumon';
 /*Query that Finds all animals with a weight between 10.4kg and 17.3kg (including the animals with the weights that equals precisely 10.4kg or 17.3kg)*/
 SELECT * FROM animals
 WHERE weight_kg BETWEEN 10.4 AND 17.3;
+/*Query for Rolling Back the species column changes*/
+BEGIN;
+
+UPDATE animals
+SET species = 'unspecified';
+
+SELECT * FROM animals;
+
+ROLLBACK;
+
+SELECT * FROM animals;
+
+
+
+
+/*Update the animals table by setting the species column to "digimon" for all animals that have a name ending in "mon":*/
+BEGIN;
+UPDATE animals
+SET species = 'digimon'
+WHERE name LIKE '%mon';
+/*UUpdate the animals table by setting the species column to "pokemon" for all animals that don't have species already set::*/
+UPDATE animals
+SET species = 'pokemon'
+WHERE species IS NULL;
+COMMIT;
```

Comment:
- [ ] Good job working on this query but please kindly include a query to verify that the change was made before committing and another the verify that the change persists after committing.